docs/CoC.md

```diff
@@ -0,0 +1,211 @@
+# Pext Code of Conduct
+
+## Purpose
+
+A primary goal of Pext is to be inclusive to the largest number of participants,
+with the most varied and diverse backgrounds possible. As such, we are committed
+to providing a friendly, safe and welcoming environment for all, regardless of
+gender, sexual and romantic orientation, ability, ethnicity, socio-economic
+status, or religion (or lack thereof).
+
+This Code of Conduct outlines our expectations for all those who participate in
+our community, as well as the consequences for unacceptable behaviour.
+
+We invite all those who participate in Pext to help us create safe and positive
+experiences for everyone.
+
+## Open culture citizenship
+
+A supplemental goal of this Code of Conduct is to increase open culture
+citizenship by encouraging participants to recognize and strengthen the
+relationships between our actions and their effects on our community.
+Communities mirror the societies in which they exist, and positive action is
+essential to counteract the many forms of inequality and abuses of power that
+exist in societies everywhere.
+
+As such, Pext is committed to the goal of open culture. Specifically, our
+community ensures that:
+
+* All source code is available under an open-source license.
+* All documentation, including artwork and logos, are available under an open
+  culture license.
+* Copyleft licenses are employed throughout to guarantee independence from
+  corporate influence and to preserve the benefits of any modifications for all.
+* Everything created by our community is available in an accessible way, without
+  DRM or other artificial restrictions to use and participation.
+
+We require all participants in our community to consider and abide by the
+principles above, in the spirit in which they are included in this Code of
+Conduct.
+
+## Expected behaviour
+
+The following behaviours are expected and requested of all community members:
+
+* Participate in an authentic and active way. In doing so, you contribute to the
+  health and longevity of this community.
+* Exercise consideration and respect in your speech and actions.
+* Attempt collaboration before conflict.
+* Refrain from demeaning, discriminatory and harassing behaviour and speech.
+* Be mindful of your surroundings and your fellow participants. Alert community
+  leaders if you notice a dangerous situation, someone in distress, or
+  violations of this Safety and Well-Being Policy, even if they seem
```

Comment:
I know you use Safety and Well-Being Policy in your version of this for your repo, but here it's called a Code of Conduct. Should probably use the term Code of Conduct here too.